app/shared/environment_variables.rb

```diff
@@ -32,6 +33,11 @@ def ci_user_password
       ENV["FASTLANE_CI_PASSWORD"]
     end
 
+    # The API token used for the bot account
+    def ci_user_api_token
+      ENV["FASTLANE_CI_BOT_API_TOKEN"]
```

Comment:
make sure you add this to the `EnvironmentVariables.md` doc 👍

Reply:
Will do 👍